library/types/test/hostname_test.rb

```diff
@@ -10,30 +10,49 @@
 
   let(:etc_hostname) { "etc.hostname.cz" }
   let(:cmd_hostname) { "cmd.hostname.cz" }
+  let(:hostname)     { Yast::Hostname }
```

Comment:
maybe easier to read is constant assignment on top of file like

```
Hostname = Yast::Hostname
```
this way it is still visible as constant and closer to real usage